Dockerfile

```diff
@@ -18,25 +21,15 @@ RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone &
     clang-tidy-14 \
     clang-format-14 \
     libpng-dev && \
+    # Intall practicerom
+    curl https://practicerom.com/public/packages/debian/pgp.pub | apt-key add - && \ 
+    echo deb http://practicerom.com/public/packages/debian staging main >/etc/apt/sources.list.d/practicerom.list &&  \ 
+    apt update && \
```

Comment:
I think we should use `apt-get` over plain `apt` on automated scripts, since `apt` is meant for user interaction.
I know this isn't something you introduced since the old docker script was doing using `apt`, but could you fix it?

Reply:
Actually, it may be better to move the practice rom setup a bit above so we can put the `practicerom-dev` dependency with the other packages

Reply:
Definitely agree with you. Changed to apt-get.

Reply:
Also took your feedback and moved it up a bit. Let me know what you think.